### PR TITLE
Refactor to use DO Catalog as embedded component in CARTO Workspace (carto 3)

### PR DESCRIPTION
**Pull request by Josmorsot**
https://app.clubhouse.io/cartoteam/story/171486/data-catalog-embed-do-catalog-as-new-section

### CODE REVIEW

**Review by VictorVelarde — APPROVED** [review]
🚀 It looks fine. Just be sure to plan a QA session coordinated with the deployment.

Remember that there are 2 PRs to create:
- this one, to apply it to CARTODB
- another PR, after publication, on PUBLIC Website pointing to public assets

**Review by VictorVelarde — CHANGES_REQUESTED** [review]
Code looks good, but CI must be green before merging

File: package.json [hunk 1]
```diff
@@ -245,7 +246,8 @@
     "build": "NODE_ENV=production webpack --progress --config webpack/v4/webpack.prod.config.js && npm run build:do-catalog",
     "build:dev": "NODE_ENV=development webpack --progress --config webpack/v4/webpack.dev.config.js",
     "build:static": "NODE_ENV=production webpack --progress --config webpack/static-pages/webpack.prod.config.js",
-    "build:do-catalog": "NODE_ENV=production vue-cli-service build --target lib --name do-catalog --formats umd-min --no-clean --inline-vue lib/assets/javascripts/do-catalog/main.js",
+    "build:do-catalog": "NODE_ENV=production vue-cli-service build --watch --target lib --name do-catalog --formats umd-min --no-clean --inline-vue lib/assets/javascripts/do-catalog/main.js",
```
Comment (Jesus89) [inline package.json:249]: I think `--watch` may not be needed here

Comment (VictorVelarde) [reply package.json:249]: true-thing

File: NEWS.md [hunk 2]
```diff
@@ -5,6 +5,7 @@ Development
 - None yet
 
 ### Features
+* Update DO Catalog dependecies and some changes to use bundle on CARTO Workspace [#16325](https://github.com/CartoDB/cartodb/pull/16325)
```
Comment (VictorVelarde) [inline NEWS.md:8]: TYPO: dependencies

**Review by VictorVelarde — APPROVED** [review]
🚀

**Comment by VictorVelarde** [comment]
PR looks good, but please link staging url in card, so @AnaDiaz85 can test also catalog there

**Pull request merged**